nifi-nar-bundles/nifi-hive-bundle/nifi-hive3-processors/src/main/java/org/apache/nifi/processors/orc/PutORC.java

```diff
@@ -63,7 +65,11 @@
                 + "the path is the directory that contains this ORC file on HDFS. For example, this processor can send flow files downstream to ReplaceText to set the content "
                 + "to this DDL (plus the LOCATION clause as described), then to PutHiveQL processor to create the table if it doesn't exist.")
 })
-@Restricted("Provides operator the ability to write to any file that NiFi has access to in HDFS or the local filesystem.")
+@Restricted(restrictions = {
+        @Restriction(
+                requiredPermission = RequiredPermission.WRITE_FILESYSTEM,
+                explanation = "Provides operator the ability to delete any file that NiFi has access to in HDFS or the local filesystem.")
```

Comment:
Is the term 'delete' correct?

Reply:
copy/paste stupid error... thanks @ijokarumawak !